app/core/middleware.py

```diff
@@ -75,6 +76,8 @@ def middleware(request: Request):
 
             patch_cache_control(response, **cache_headers)
 
+        patch_vary_headers(response, ['Accept', 'Accept-Encoding'])
```

Comment:
Is Django handling compression? (Normally done at Nginx etc?)

Reply:
That would explain it then, seemed strange. Will remove.